execute/requirements.txt

```
@@ -0,0 +1,4 @@
+flatbuffers
+numpy
+jsonschema>=4.6.2
+git+https://github.com/SonySemiconductorSolutions/aitrios-sample-application-keypoint-detection@v1.0.1#egg=console-access-api&subdirectory=backend/lib/console-access-api
```

It's not required but I like to specify the project name when using a VCS requirement

```suggestion
console-access-api @ git+https://github.com/SonySemiconductorSolutions/aitrios-sample-application-keypoint-detection@v1.0.1#egg=console-access-api&subdirectory=backend/lib/console-access-api
```

Very fancy, nicely done. Any reason to prefer `git+https` over `git+ssh` here? I find that the former is more convenient on Windows (getting SSH auth to work is mildly annoying there) and the latter is more convenient on Linux. No strong preference, but I usually write `git+ssh` and deal with it on Windows when I have to.